_data/projects/leaflet.yml

```diff
@@ -0,0 +1,10 @@
+name: Leaflet
+desc: JavaScript library for mobile-friendly interactive maps
+site: http://leafletjs.com
+tags:
+- Javascript
+- Maps
+- GIS
+upforgrabs:
+  name: easy-fix help-wanted
```

Comment:
Yeah, it's been a long time since the site content had a good refresh - I need to sit down and rework the initial vision we had for this project. Our target audience is mainly people who want to get started contributing to a project, but the "needs specific environment" isn't something I've actively thought about.

> Shall I amend this PR?

Yes please!